engines/order_management/app/services/order_management/stock/package.rb

```diff
@@ -101,7 +101,7 @@ def shipping_methods
 
       def inspect
         out = "#{order} - "
-        out << contents.map do |content_item|
+        out.dup << contents.map do |content_item|
```

Comment:
looking at this again, it looks broken:
![image](https://user-images.githubusercontent.com/1640378/98661660-a948d800-233e-11eb-90b8-168ec821eb5e.png)

Reply:
This change wasn't actually related to an error in the build, but when I tried debugging a package object in the console with `#inspect` I got fatal errors due to "modifying a frozen string". Here we have the string `out = "#{order} - "` and then add to the string with `out <<`, but we're using **frozen string literals** in this class, so this operation is not valid, right?

Reply:
But yes, it looks like we need to tweak this a bit.

Reply:
Fix and regression test added :+1:

Reply:
ah, yes, just remove the last line!
thanks :+1: